assets/stylesheets/sections/_notifications.scss

```diff
@@ -2,6 +2,8 @@
  * Notifications
  */
 
+@import 'notifications/style';
```

Comment:
Just commenting here so that we don't lose it, but we should look at breaking this file into the different component pieces.